wgsl/index.bs

```diff
@@ -11718,10 +11988,11 @@ reduce a shader's memory bandwidth demand.
 
 <table class='data'>
   <thead>
-    <tr><td>Overload<td>Description
+    <tr><td>Parameterization<td>Overload<td>Description
```

Comment:
I'm not a huge fan of the asymmetry here between pack and unpack built-in functions. I'd suggest either removing the f16 versions of packing or rename all the functions similarly to MSL, e.g.:
```
pack4x8snormf16
pack4x8snormf32
unpack4x8snormf16
unpack4x8snormf32
```

Reply:
Remove the f16 version of packing.